src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.tsx

```diff
@@ -47,19 +52,28 @@ export const AssetManagementForm = ({
   isSurrendered,
   onApproveNewTransferTargets,
   approveNewTransferTargetsState,
+  onTransferToNewEscrow,
+  transferToNewEscrowState,
 }: AssetManagementFormProps) => {
   const isHolder = account === holder;
   const isBeneficiary = account === beneficiary;
   const canSurrender = isBeneficiary && isHolder;
   const canEndorseBeneficiary = isBeneficiary && isHolder;
   const canNominateBeneficiaryHolder = isBeneficiary && !isHolder;
+  const canEndorseTransfer =
+    !!approvedBeneficiary &&
+    approvedBeneficiary !== "0x0000000000000000000000000000000000000000" &&
+    !!approvedHolder &&
+    approvedHolder !== "0x0000000000000000000000000000000000000000" &&
+    isHolder;
```

Comment:
can be 

```suggestion
const canEndorseTransfer = approvedBeneficiary && approvedHolder && isHolder;
```

yeah?

Reply:
ya i did this at first but by default if there is no nomination, it will be "0x00000..." then the dropdown will appear.

Reply:
ah ok i see it now